userapi/src/main/java/fk/prof/userapi/http/UserapiApiPathConstants.java

```diff
@@ -1,13 +1,31 @@
 package fk.prof.userapi.http;
 
 public final class UserapiApiPathConstants {
+    private UserapiApiPathConstants() {
+    }
 
-  private UserapiApiPathConstants() { }
+    public static final String POLICY_API_PREFIX = "/policy";
+    public static final String LIST_POLICY_API_PREFIX = "/list/policy";
+    public static final String DELIMITER = "/";
+    public static final String APPS_PREFIX = "/apps";
+    public static final String CLUSTERS_PREFIX = "/clusters";
+    public static final String PROCS_PREFIX = "/procs";
+    public static final String PROFILES_PREFIX = "/profiles";
+    public static final String PROFILE_PREFIX = "/profile";
+
+    public static final String APPS = APPS_PREFIX;
```

Comment:
Just like app,cluster,proc listings for policy, app,cluster,proc listings here should be prefixed with "/profiles"

Reply:
Added "/profiles" prefix.